Source/FolioReaderCenter.swift

```diff
@@ -8,7 +8,9 @@
 
 import UIKit
 //import ZFDragableModalTransition
-
+public extension Notification.Name {
+    static let pageDidLoadNotif = Notification.Name("pageDidAppearNotif")
```

Comment:
`pageDidLoadNotification` sounds about right to me, maybe we don't need the abbreviation.

Reply:
name changed to pageDidLoadNotification